Month1/Week2/Section1/Exercise4/Main4.cpp

```diff
@@ -27,26 +27,28 @@ class FPrinter
 {
 private:
 	std::string OutputString;
+	
 public:
 	FPrinter()
 	{
+		std:: string OutputString = "";
 	}
 	FPrinter(std::string EntryString)
 	{
-		EntryString = "";
+		OutputString = EntryString;
 	}
 	
 	void InputString() 
 	{
 		std::string EntryString;
 		std::cout << "Welcome! Please enter a string:" << std::endl;
-		std::getline( std:: cin, EntryString);
-		while(!std::cin.good())
+		std::getline(std::cin, EntryString);
+		while(!std::cin.good()||EntryString==" ")
```

Comment:
Mind your spaces here. Remember that operators should have spaces between them except the `!`, `;`, `::`, and `.` operators.

Reply:
Ok  change the spaces